### ISSUE TITLE
opening inspector errors with "Cannot read property 'hero' of undefined"

### ISSUE THREAD

**Issue by jmhodges** [mtgatracker/mtgatracker#464]
I just installed 5.0.2. I've never had an install of it before and do not have an Inspector web account.

When I open the inspected from the drop down menu, it opens an alert "TypeError: Cannot read property 'hero' of undefined'". That occurs whenever I click "Home"

If I click Profile, I get an uncaught javascript exception: "Cannot read property 'date' of undefined". I'm not able to copy out the stack trace (but there is on this one) and points to the inspectorApi.js file and the last three characters I can see in the filepath are ":33". The window can't be scaled to let me see the elided bits between that ".js" and those final characters, so I'm not sure if that's the line number or the column number.

I see that the inspector is now local-only, but there doesn't seem to be a way to do the equivalent of a login. (In the top right, there is a "Hello, unknown" dropdown but it just has a setting for "Dark Mode" in it)

**Comment by smkiewel** [comment]
Have you finished a game with the tracker running? Sounds like there is a bug where it assumes there are games and is trying to read nonexistent data.

**Comment by jmhodges** [comment]
Hm, I just played a game and things seem to have gotten more broken, unfortunately.

The app doesn't load correctly. I've attached a screenshot. Hrm.

![capture](https://user-images.githubusercontent.com/598/49689965-182baf80-fade-11e8-9098-8a9257bac8ba.PNG)

**Comment by jmhodges** [comment]
(Note: that screenshot is from after I closed and reopened it because it seemed to freeze after the game)

**Comment by smkiewel** [comment]
Yes, there is a bug where the menu doesn't show in rollup mode. I'm trying to fix it, but it's really difficult. But it is tracked in #454.

**Comment by jmhodges** [comment]
ty!
